patches/@supabase+supabase-js+1.35.7.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/node_modules/@supabase/supabase-js/dist/main/SupabaseClient.js b/node_modules/@supabase/supabase-js/dist/main/SupabaseClient.js
+index 8d78eae..5025637 100644
+--- a/node_modules/@supabase/supabase-js/dist/main/SupabaseClient.js
++++ b/node_modules/@supabase/supabase-js/dist/main/SupabaseClient.js
+@@ -57,14 +57,15 @@ class SupabaseClient {
+         this.realtimeUrl = `${_supabaseUrl}/realtime/v1`.replace('http', 'ws');
+         this.authUrl = `${_supabaseUrl}/auth/v1`;
+         this.storageUrl = `${_supabaseUrl}/storage/v1`;
+-        const isPlatform = _supabaseUrl.match(/(supabase\.co)|(supabase\.in)/);
+-        if (isPlatform) {
+-            const urlParts = _supabaseUrl.split('.');
+-            this.functionsUrl = `${urlParts[0]}.functions.${urlParts[1]}.${urlParts[2]}`;
+-        }
+-        else {
+-            this.functionsUrl = `${_supabaseUrl}/functions/v1`;
+-        }
++        this.functionsUrl = `${_supabaseUrl}/functions/v1`
```

Comment:
We need to use this URl as it seems to work better for users. However, this path is only set this way in V2 of the library.